app/views/layouts/application.html.erb

```diff
@@ -8,21 +8,16 @@
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+
   </head>
 
   <body>
     <div class="container">
-        <ul class="nav nav-tabs">
-          <% if session['procore_token'] %>
-            <li><%= link_to "Home", users_home_path %></li>
-            <li><%= link_to "Me", users_me_path %></li>
-            <li><%= link_to "Sign Out", login_revoke_path %></li>
-          <% else %>
-            <li><%= link_to "Sign In", login_index_path %></li>
-          <% end %>
-        </ul>
+      <%= render "layouts/nav_bar" %>
+      <%= render "layouts/flash_message" %>
```

Comment:
Rails normally refers to `layouts` as something that "wraps around" the content to display.

For instance, _this file_ is a layout because it renders the head and then `<%= yield %>` to the content.

Instead, portions of HTML rendered inside another view are called `partials`.

So I would suggest renaming these files to partials.

Reply:
They aren't layouts, but they're also only used by `application.html.erb` which is a layout. So I figure it's logical to put them in the same folder, like you might put partials next to `/users` if they were only used on that page. Partials are prefixed by an `_` to distinguish them.

I'll also point out Dev Portal puts some partials in `/layouts` and doesn't have a `/partials` directory. Doesn't make it right though 😉